web/src/tippyjs.ts

```diff
@@ -178,6 +182,88 @@ export function initialize(): void {
+                                    "{display_count, plural, one {You have {display_count} unread message.} other {You have {display_count} unread messages.}}",
+                            },
+                            {display_count},
+                        ),
+                    );
+                    break;
+                case "mentions":
+                    display_count = unread.unread_mentions_counter.size;
+                    $container.find(".views-message-count").text(
+                        $t(
+                            {
+                                defaultMessage:
+                                    "{display_count, plural, one {You have {display_count} unread mention.} other {You have {display_count} unread mentions.}}",
+                            },
+                            {display_count},
+                        ),
+                    );
+                    break;
+                case "starred_message":
+                    display_count = starred_messages.get_count();
+                    $container.find(".views-message-count").text(
+                        $t(
+                            {
+                                defaultMessage:
+                                    "{display_count, plural, one {You have {display_count} starred message.} other {You have {display_count} starred messages.}}",
+                            },
+                            {display_count},
+                        ),
+                    );
+                    break;
+                case "drafts":
+                    display_count = drafts.draft_model.getDraftCount();
+                    $container.find(".views-message-count").text(
+                        $t(
+                            {
+                                defaultMessage:
+                                    "{display_count, plural, one {You have {display_count} draft.} other {You have {display_count} drafts.}}",
+                            },
+                            {display_count},
+                        ),
+                    );
+                    break;
+                case "scheduled_message":
+                    display_count = scheduled_messages.get_count();
+                    $container.find(".views-message-count").text(
+                        $t(
+                            {
+                                defaultMessage:
+                                    "{display_count, plural, one {You have {display_count} scheduled message.} other {You have {display_count} scheduled messages.}}",
+                            },
+                            {display_count},
+                        ),
+                    );
+                    break;
+            }
+
+            // Since the tooltip is attached the anchor tag which doesn't
+            // include with of the ellipsis icon, we need to offset the
+            // tooltip so that the tooltip is displayed to right of the
+            // ellipsis icon.
```

Comment:
I think you meant:
- attached *to an* anchor ...
- include *width* of ...

Reply:
Fixed the comment